tests/requirements.txt

```diff
@@ -1,2 +1,6 @@
 docker-compose==1.28.5
+flask-oidc==1.4.0
```

Comment:
How did you determine this subset of dependencies?

Reply:
I invoked tests multiple and added the packages one by one.

Reply:
Shouldn't this just also include the requirements from `requirements.txt`?

Reply:
Maybe,  I didn't spend to much thoughts on that. We might want to switch to pipenv or poetry.

Reply:
Could you please only include the dependencies not in `requirements.txt`?

Reply:
Done